chapter_recurrent-modern/seq2seq_vn.md

```diff
@@ -152,7 +158,8 @@ Since an LSTM is used, the `state` list will contain both the hidden state and t
 However, if a GRU is used, the `state` list will contain only one element---the hidden state in the final timestep with shape (number of hidden layers, batch size, number of hidden units).
 -->
 
-*dịch đoạn phía trên*
+Khi LSTM được sử dụng, danh sách `state` sẽ chứa cả trạng thái ẩn và ô nhớ với cùng kích thước (số lượng tầng ẩn, kích thước batch, số lượng nút ẩn).
```

Comment:
```suggestion
Vì LSTM đang được sử dụng, danh sách `state` sẽ chứa cả trạng thái ẩn và ô nhớ với cùng kích thước (số lượng tầng ẩn, kích thước batch, số lượng nút ẩn).
```
`Since` chỉ trong đoạn code chương này.